engine/apps/api/views/user.py

```diff
@@ -219,10 +221,24 @@ class UserView(
         "^username",
         "^slack_user_identity__cached_slack_login",
         "^slack_user_identity__cached_name",
+        "^teams__name",
```

Comment:
one of the new requirements for the user search functionality in the Direct Paging dialog box will be to search for users based on the teams they currently belong to